platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r813/ProblemProgressEventCrossVersionTest.groovy

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 the original author or authors.
+ * Copyright 2025 the original author or authors.
```

Comment:
🤔 I don't think we should change the year in existing file. There are a couple changes like this in the PR changeset

Reply:
I ran an update copyright task in IJ, as some files were without it, and it broke `sanityCheck`.
It seems it was a bit too eager. Interestingly, only in this and `DelegatingProblemBuilder.java`'s case.